packages/popover/src/index.js

```diff
@@ -0,0 +1,246 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import { Manager, Reference, Popper } from 'react-popper'
+import { Box, Flex, CloseButton, theme } from 'pcln-design-system'
+import styled from 'styled-components'
+import DEFAULTS_MODIFIERS from './helpers/defaultModifiers'
+
+const defaultProps = {
+  theme: theme,
+  p: 2,
+  bg: 'white',
+  placement: 'top'
+}
+class PopOver extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isPopOverOpen: false
+    }
+
+    this.contentRef = React.createRef()
+    this.triggerRef = React.createRef()
+    this.handleToggle = this.handleToggle.bind(this)
+    this.handleOpen = this.handleOpen.bind(this)
+    this.handleClose = this.handleClose.bind(this)
+    this.setFocusToRef = this.setFocusToRef.bind(this)
+    this.calcOffset - this.calcOffset.bind(this)
```

Comment:
Yep, I will be updating this once I'm done adding whatever methods I need. I like to use this pattern initially, just because it gives me a concise list of all the functions I currently have :)